web/src/components/profile/ProfileErrorAlert.tsx

```diff
@@ -0,0 +1,61 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { ProfileContentField } from "@/lib/types/types";
+import { templateEval } from "@/lib/utils/helpers";
+import { ReactElement } from "react";
+
+interface Props {
+  fieldErrors: string[];
+}
+
+export const ProfileErrorAlert = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  const displayProfileErrorAlert = (): boolean => props.fieldErrors.length > 0;
+
+  const getHeader = (field: string): string => {
```

Comment:
[Dust] would a variable name like `getLabel` be better? We are not grabbing the header.